examples/siro_sandbox/sandbox_app.py

```diff
@@ -57,6 +60,67 @@ def wrapper(*args, **kwds):
     return wrapper
 
 
+class SandboxState(Enum):
+    SIMULATION = 1
```

Comment:
Nit: how about a better name than simulation? I guess it's the phase where the user can control the humanoid to complete the task.

Btw, I understand you are not stepping the simulation during the tutorial, hence the motivation to call this state "simulation". My point is really that we'll likely have other states in the future besides tutorial and this one.